code/manager/.storybook/preview.jsx

```diff
@@ -0,0 +1,248 @@
+import global from 'global';
```

Comment:
this is just copied from the old official sb example - I only removed the header warning for missing envs.